submodules/wizard/wizard.js

```diff
@@ -583,6 +588,15 @@ define(function(require) {
 							fullName: representativeFullName
 						};
 					}
+
+					arePropertiesEmpty(accountContactsData.salesRep);
+				}
+
+				arePropertiesEmpty(accountContactsData.technicalContact);
+				arePropertiesEmpty(accountContactsData.billingContact);
+
+				if (_.every(accountContactsData, 'isEmpty')) {
+					_.merge(accountContactsData, { sectionEmpty: true });
```

Comment:
Same here, `lodash#merge` may be adding unnecessary overhead, because only one property is set.
```suggestion
					accountContactsData.sectionEmpty = true;
```